packages/preview/modern-acad-cv/0.1.0/template/example.typ

```diff
@@ -0,0 +1,109 @@
+// #import "modern-acad-cv.typ": *
+#import "@local/modern-acad-cv:0.1.0": *
```

Comment:
Could you use the `@preview` import here? This file will be copied as is when creating a project from your template, and these imports won't work by default.

```suggestion
#import "@preview/modern-acad-cv:0.1.0": *
```

Reply:
Ah thanks for the quick check. I'll update and resend.